doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -162,5 +147,31 @@ To see stopped or misbehaving nodes following command can be useful:
 mongooseimctl mnesia info | grep "stopped db nodes"
 ```
 
-This command shows which nodes are considered stopped. 
+This command shows which nodes are considered stopped.
 This does not necessarily indicate that they are down but might be a symptom of a communication problem.
+
+## Load Balancing
+
+### Elastic Load Balancer (ELB)
+
+When using ELB pleas be advised that some worm-up time maybe needed before
```

Comment:
When using ELB please be advised that some warm-up time may be needed before the load balancer works efficiently for a big load.